sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 2.1.0-beta.1 (Unreleased)
+## 2.0.1 (2021-01-11)
+
+### New Features
+
+- Changed connections from sharing the global `Schedulers.single()` to having a `Scheduler.newSingle()` per connection.
```

Comment:
@conniey @srnagar @hemanttanwar

Reply:
per connection ->  `per connection to **improve performance`**